pyproject.toml

```diff
@@ -0,0 +1,66 @@
+[build-system]
+requires = [ "setuptools>=61", "wheel", "setuptools-git-versioning>=2.0,<3", ]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools-git-versioning]
+enabled = true # Allows setuptools-git-versioning to pull version from git state
+
+[tool.setuptools]
+packages = ["model_interpreter"]
+
+[project]
+name = "model-interpeter"
```

Comment:
B: typo - 'model-interpeter' should be 'model-interpreter'

Reply:
updated here https://github.com/lvgig/model_interpreter/pull/2/commits